01-start/03-begin.html.md

```diff
@@ -27,12 +27,13 @@ Before we get hacking away on content, we have to create our project and set it
 ``` bash
 mkdir my-new-website
 cd my-new-website
+docpad --offline init
```

Comment:
The primary use is to work around the DocPad bug that causes the whole process to fail if you're initializing in a directory that's already part of a Git repo. I'd be happy to sit down and help you build a test case to demonstrate this (and whatever bits of new test framework we need to support such a test case); grab me on the Bevry Slack if you'd like to look at this further.